addons/account/wizard/accrued_orders.py

```diff
@@ -105,28 +92,16 @@ def _get_computed_account(self, order, product, is_purchase):
         else:
             return accounts['income']
 
-    def _compute_move_vals(self):
-        def _get_aml_vals(order, balance, amount_currency, account_id, label="", analytic_distribution=None):
-            if not is_purchase:
-                balance *= -1
-                amount_currency *= -1
-            values = {
-                'name': label,
-                'debit': balance if balance > 0 else 0.0,
-                'credit': balance * -1 if balance < 0 else 0.0,
-                'account_id': account_id,
-            }
-            if analytic_distribution:
-                values.update({
-                    'analytic_distribution': analytic_distribution,
-                })
-            if len(order) == 1 and self.company_id.currency_id != order.currency_id:
-                values.update({
-                    'amount_currency': amount_currency,
-                    'currency_id': order.currency_id.id,
-                })
-            return values
+    def _get_aml_vals(self, label, balance, account_id, amount_currency=None, analytic_distribution=None, order=None, is_purchase=None, **kwargs):
```

Comment:
```suggestion
    def _get_aml_vals(self, label, balance, account_id, amount_currency=None, analytic_distribution=None, order=None, is_purchase=None):
```

`**kwargs` doesn't seem to be used.